salt/modules/yumpkg.py

```diff
@@ -1056,9 +1056,10 @@ def install(name=None,
     return ret
 
 
-def upgrade(refresh=True, fromrepo=None, skip_verify=False, **kwargs):
+def upgrade(name=None, refresh=True, fromrepo=None, skip_verify=False, pkgs=None, normalize=True, **kwargs):
```

Comment:
I agree that this is a strong argument, but perhaps we could find a middle road and simply warn in the case of potentially dangerous behaviour and put a deprecation warning stating that we'll be moving this arguments in an upcoming release?